### DISCUSSION TITLE
Feature request: Build Server Protocol support

### DISCUSSION THREAD

**Discussion by Zeioth** [Zeioth/compiler.nvim#18] [Ideas]
This discussion is a continuation of [this feature request](https://github.com/Zeioth/compiler.nvim/issues/17) started by [616b2f](https://github.com/616b2f).

**Comment by 616b2f** [comment]
I started to experiment with the client implementation of bsp for neovim, the idea would be to have it implemented as in lib that does not provide any user interface, similar like dap.nvim does.
